icxcli/icx/wallet/__init__.py

```diff
@@ -141,8 +141,8 @@ def transfer_value_with_the_fee(password, fee, to, amount, file_path, url):
 
         method = 'icx_sendTransaction'
 
-        amount = int(amount)
-        fee = int(fee)
+        amount = int(float(amount))
```

Comment:
Please use ```is_integer()``` function. See the link below. https://stackoverflow.com/questions/21583758/how-to-check-if-a-float-value-is-a-whole-number

And need to show the error message 'Amount and fee should be integer and loop unit, not icx' when user enter floating point for amount and fee.